src/patternfly/components/Drawer/examples/Drawer.md

```diff
@@ -169,14 +168,27 @@ import './Drawer.css'
 {{/drawer}}
 ```
 
-### Modified panel padding
+### Modified panel body padding
```

Comment:
Is this a valid use of the pf-m-padding class that we want to showcase? I updated the example to show one way to use the class on panel body content (previously this example wasn't any different from a basic drawer example), but in React we don't have a prop to apply the class on a DrawerPanelBody (though the class could be manually passed in if a consumer knew it could be used).